src/components/group-settings.jsx

```diff
@@ -1,57 +1,58 @@
 import React from 'react'
-import {Link} from 'react-router'
-import {connect} from 'react-redux'
+import { Link } from 'react-router'
+import { connect } from 'react-redux'
 import _ from 'lodash'
 
-import {updateGroup} from '../redux/action-creators'
+import { updateGroup } from '../redux/action-creators'
 import GroupSettingsForm from './group-settings-form'
 import throbber100 from 'assets/images/throbber.gif'
 
 const GroupSettings = (props) => (
-  props.groupSettings.status === 'loading' ? (
-    <div className="box">
-      <div className="box-header-timeline">
-        Group settings
-      </div>
-      <div className="box-body">
-        <img width="100" height="100" src={throbber100}/>
-      </div>
+props.groupSettings.status === 'loading' ? (
+  <div className="box">
+    <div className="box-header-timeline">
+      Group settings
+    </div>
+    <div className="box-body">
+      <img width="100" height="100" src={throbber100} />
     </div>
+  </div>
   ) : props.groupSettings.status === 'success' ? (
     <div className="box">
       <div className="box-header-timeline">
-        <Link to={`/${props.group.username}`}>{props.group.username}</Link>: group settings
+        <Link to={`/${props.group.username}`}>
+        {props.group.username}
+        </Link>: group settings
       </div>
       <div className="box-body">
-        <GroupSettingsForm
-          group={props.group}
-          updateGroup={props.updateGroup}
-          {...props.groupSettingsForm}/>
+        <GroupSettingsForm group={props.group} updateGroup={props.updateGroup} {...props.groupSettingsForm}/>
       </div>
     </div>
-  ) : props.groupSettings.status === 'error' ? (
-    <div className="box">
-      <div className="box-header-timeline">
-        Group settings
+    ) : props.groupSettings.status === 'error' ? (
+      <div className="box">
+        <div className="box-header-timeline">
+          Group settings
+        </div>
+        <div className="box-body">
+          <div className="alert alert-danger">
+            {props.groupSettings.errorMessage}
+          </div>
+        </div>
       </div>
-      <div className="box-body">
-        <div className="alert alert-danger">{props.groupSettings.errorMessage}</div>
-      </div>
-    </div>
-  ) : (
-    <div/>
-  )
+      ) : (
+      <div/>
+      )
 )
 
-function mapStateToProps(state) {
+function mapStateToProps (state, ownProps) {
   return {
-    group: (_.find(state.users, { 'username': state.router.params.userName }) || {}),
+    group: (_.find(state.users, { 'username': ownProps.params.userName }) || {}),
     groupSettings: state.groupSettings,
     groupSettingsForm: state.groupSettingsForm
   }
 }
 
-function mapDispatchToProps(dispatch) {
+function mapDispatchToProps (dispatch) {
```

Comment:
Let's move all the style changes to the separate commit. (And I'd say we don't want this one, actually.)